### ISSUE TITLE
Feature suggestions： Exclude class name or package name

### ISSUE THREAD

**Issue by ohyeah521** [secure-software-engineering/FlowDroid#784]
Does it support setting excluded class names or package names? Because some large Android applications have a lot of code from third-party SDKs, and these third-party SDKs are not practical when studying the security of the APK's own code.

**Comment by StevenArzt** [comment]
Yes, that is possible. If you initialize Soot on your own, you can mark the classes to exclude as "library classes" in Soot's terminology. In the ``InfoflowConfiguration``, you then have the flag ``excludeSootLibraryClasses`` to ensure that FlowDroid doesn't touch them.

**Comment by ohyeah521** [comment]
@StevenArzt   what should I do if I use the command line version soot-infoflow-cmd-2.13.0-jar-with-dependencies.jar?

**Comment by StevenArzt** [comment]
The command-line application does not support such exclusions.
